scripts/env.d/vars

```diff
@@ -1,6 +1,7 @@
 export DEVENVFLAVOR=""
 export DEVENVPREFIX=""
 export DEVENVPATH_BACKUP="${PATH}"
+export DEVENVDLROOT="${DEVENVROOT}/var/downloaded"
```

Comment:
Add a new variable for the global location for tarball archive.  (It may be used for other data in the future.)